apps/tailwind-components/components/input/Ref.vue

```diff
@@ -241,5 +243,5 @@ function loadMore() {
       load {{ entitiesLeftToLoad }} more
     </ButtonText>
   </template>
-  <ButtonText v-else>No results found</ButtonText>
+  <ButtonText v-else :inverted="inverted">No results found</ButtonText>
```

Comment:
right, met dit inverted moeten we nog aan de gang. David was optimistisch dat het op tailwind nivo kan.